### DISCUSSION TITLE
Custom actions on edit modal in relationmanager

### DISCUSSION THREAD

**Discussion by lucacastelnuovo** [filamentphp/filament#6548] [Help] [answered]
Is it possible to add a custom action (for example deletion) on relationmanager edit modals?

![SCR-20230518-knne](https://github.com/filamentphp/filament/assets/26206253/d358ddac-8d03-4a1d-b982-1268fdc321bf)

**Comment by danharrin** [comment, marked as answer]
Not at the moment, it is possible in v3 though.
